src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -135,21 +119,30 @@ private boolean pmwNewVendorRequestProcessingIntoKfsWasSuccessful(PaymentWorksVe
     
     private boolean canPaymentWorksNewVendorRequestProcessingContinueForVendor(PaymentWorksVendor stgNewVendorRequestDetailToProcess, PaymentWorksNewVendorRequestsBatchReportData reportData) {
         List<String> errorMessages = new ArrayList<String>();
-        if (pmwDtosCouldConvertCustomAttributesToPmwJavaClassAttributes(stgNewVendorRequestDetailToProcess) &&
-            pmwNewVendorAttributesConformToKfsLengthsOrFormats(stgNewVendorRequestDetailToProcess, errorMessages) &&
-            allPmwNewVendorIsoCountriesMapToSingleFipsCountry(stgNewVendorRequestDetailToProcess, errorMessages) &&
-            pmwNewVendorIdentifierDoesNotExistInKfsStagingTable(stgNewVendorRequestDetailToProcess, errorMessages)){
+        if (pmwNewVendorNotNull(stgNewVendorRequestDetailToProcess, errorMessages)
+                && pmwDtosCouldConvertCustomAttributesToPmwJavaClassAttributes(stgNewVendorRequestDetailToProcess)
+                && pmwNewVendorAttributesConformToKfsLengthsOrFormats(stgNewVendorRequestDetailToProcess, errorMessages)
+                && allPmwNewVendorIsoCountriesMapToSingleFipsCountry(stgNewVendorRequestDetailToProcess, errorMessages)
+                && pmwNewVendorIdentifierDoesNotExistInKfsStagingTable(stgNewVendorRequestDetailToProcess, errorMessages)) {
             return true;
-        }
-        else {
-            if (!errorMessages.isEmpty()){
+        } else {
+            if (!errorMessages.isEmpty()) {
                 reportData.getRecordsThatCouldNotBeProcessedSummary().incrementRecordCount();
                 reportData.addPmwVendorThatCouldNotBeProcessed(new PaymentWorksBatchReportRawDataItem(stgNewVendorRequestDetailToProcess.toString(), errorMessages));
```

Comment:
This line assumes that the vendor object is non-null, but the new null-vendor check will allow this line to be called with a null vendor since it updates the error messages list. Some adjustments are needed in order to avoid a NullPointerException.